tests/scripts/ci.py

```diff
@@ -178,6 +213,8 @@ def docker(name: str, image: str, scripts: List[str], env: Dict[str, str]):
         clean_exit(f"Error invoking Docker: {e}")
     except KeyboardInterrupt:
         cmd(["docker", "stop", "--time", "1", name])
+    finally:
+        script_file.unlink()
```

Comment:
should we have a debug mode here which doesn't remove the script? or is the idea you debug it anyway via the trap above?

Reply:
added a guard for an env variable `DEBUG=1` to keep it in